scripts/smoke_test.sh

```diff
@@ -0,0 +1,258 @@
+#!/bin/bash
+
+if [[ $1 == "" ]]; then
+  echo "Missing test image!"
+  exit 1
+fi
+
+image_name=$(basename $1)
+echo "=== Smoke testing ${image_name}"
+
+[[ $image_name =~ ^(.+)-(libvirt|ovirt)-([0-9]*)([a-z]*)(.+)\.qc2$ ]]
```

Comment:
Minor, but I tend to wrap variables in `""` so that if they are blank you don't end up with a weird syntax error.  So, here, something like

```suggestion
[[ "$image_name" =~ ^(.+)-(libvirt|ovirt)-([0-9]*)([a-z]*)(.+)\.qc2$ ]]
```

Reply:
I agree. Fun fact: the rules around `[` aka `test` and `[[` are different here:

```bash
$ [[ $x = "1" ]] && echo "yay"
$ [ $x = "1" ] && echo "yay"
-bash: [: =: unary operator expected
```